moja-global/FLINT-UI
# Updated TablePoint.vue file to use Ant Design Tables and Added Reusab…

Fixes #248 

Updated TablePoint.vue file to use Ant Design Tables.
Renamed TablePoint.vue to Table.vue as it's reusable table component.
Added PointOuterTable.vue file to allow switching between graph and table
Added Table and graph on configuration page itself as suggested in the Figma design.

Testing
1. Explore the Point page from the homepage of flint-ui
2. Click on run to load the point data.
3. Click on the Point Output tabel to see Table and Graph.
